_data/governance.yml

```diff
@@ -0,0 +1,37 @@
+introText: This section is updated every 6 months by the Scala Center.
+cards:
+  - title: "Scala Improvement Process"
+    description: "The SIP is the primary mechanism for evolving the Scala language."
+    expandText: "Process & People"
+    links:
+      - name: SIP Committee Members
+        url: https://docs.scala-lang.org/sips/process-specification.html#the-sip-committee
+      - name: SIP Process Specification
+        url: https://docs.scala-lang.org/sips/process-specification.html
+  - title: "Scala 3 Maintenance"
+    description: "Managing the Scala 3 compiler, standard library and documentation."
+    expandText: "Process & People"
+    links:
+      - name: Core Maintainers
+        url: https://github.com/lampepfl/dotty/blob/main/MAINTENANCE.md#compiler
+      - name: Maintenance Procedures
+        url: https://github.com/lampepfl/dotty/blob/main/MAINTENANCE.md
+  - title: "Tooling"
+    description: "Managing build tools, linters, IDEs, and other tools."
+    expandText: "Process & People"
+    links:
+      - text: "Process and People are TBD, please stay tuned! However you can read below about the Scala Tooling Summit:"
```

Comment:
I think this section should be filled before we merge the PR.

Reply:
The current status on this is that we've only conducted one Tooling Summit during which no process was adopted. Changing this might be the goal for the next Summit in September.

Reply:
Even if there is no “process”, I would expect the Governance section to give more understanding to the readers about which organizations do exist and actively work on the tooling ecosystem. I think we should mention at least VirtusLab with Scala CLI and Metals, and Jetbrains with IntelliJ.